README.md

```diff
@@ -179,6 +180,7 @@ R2U.ar.attach({
 | `sku`                          | product SKU                                                                                  | `''`                 |
 | `event`                        | event that triggers AR                                                                       | `'click'`            |
 | `resize`                       | Option to resize 3D model on AR experience                                                   | `false`              |
+| `showInstructions`             | Option to enter instructions in the AR                                                       | `false`              |
```

Comment:
Acho que a descrição não explica bem o que a feature faz. Pensa em um usuário que não entende do SDK lendo isso. Entrar nas instruções em AR? Entrar como? Que instruções? 

Sugestão: `When true, shows an image in full-screen view explaining how to place and manipulate a 3D object on AR before proceeding to the camera experience`